File: librarian-app/src/main/java/com/github/dkurata38/liblio/librarian_app/LibrarianApplication.java
package com.github.dkurata38.liblio.librarian_app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;
import com.github.dkurata38.liblio.application.ApplicationConfig;
import com.github.dkurata38.liblio.infrastructure.InfrastructureConfig;

@SpringBootApplication
@Import({ApplicationConfig.class, InfrastructureConfig.class})
public class LibrarianApplication {
    public static void main(String[] args) {
        SpringApplication.run(LibrarianApplication.class, args);
    }
}

File: librarian-app/src/main/kotlin/com/github/dkurata38/liblio/librarian_app/LibrarianApplication.kt
package com.github.dkurata38.liblio.librarian_app

import com.github.dkurata38.liblio.application.ApplicationConfig
import com.github.dkurata38.liblio.infrastructure.InfrastructureConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(ApplicationConfig::class, InfrastructureConfig::class)
class LibrarianApplication

fun main(args: Array<String>) {
    runApplication<LibrarianApplication>(*args)
}
